app/models/section.rb

```diff
@@ -35,6 +36,12 @@ def to_param
     position.to_s
   end
 
+  delegate :count, to: :chapters, prefix: true
+
+  # TODO: Method deprecated!
+  # Why: ".to_s" should not be use to represent any
```

Comment:
Personally I disagree with this - debugging, and 'inspecting' should be handled by `.inspect` - its there for that exact use case?

I'm not a fan of overriding `.to_s`, but in general it should be the 'stringified' version of an object - which may well be just the name ?

Reply:
So I didn't have strong views. I basically never use either since in pry you can just do `ls <object reference name>` and get a view on the internal state of the object

Reply:
```ruby
section = ... # code to get a section

link_to "Section: #{section}"
```